jcl/src/java.base/share/classes/jdk/internal/vm/Continuation.java

```diff
@@ -206,6 +206,7 @@ public final void run() {
 				/* Continuation yielded */
 			}
 			JLA.setContinuation(carrier, parent);
+			unlockAccess();
```

Comment:
Didn't equate `isAccessible = true` to unlock. For consistency, we should the use the API instead of directly updating the field from different places. So, removed the extra unlock. Also, the unlock should happen at the end once the carrier is reset. Otherwise, it may cause a race condition.